src/components/TagGeneratorInstructions.js

```diff
@@ -1,159 +1,101 @@
-  const classes = useStyles()
-
-  const PaperComp = ({ text,btext,bhref }) =>{
-    return (
-      <Grid item className={classes.gridStyle} >
-        <Paper variant="outlined" className={classes.paper}>
-          <Grid style={{ padding:'16px' }}>
-            <Typography variant='body1' className={classes.ptextStyle}>{text}</Typography>
-          </Grid>
-          <Grid><Button href={bhref} className={classes.btnColor}>{btext}</Button></Grid>
-        </Paper>
-      </Grid>
-    )
-  }
-  return (
-    <>
-      <Box className='containerWhite'>
-        <Container>
-          <Grid container direction="column" alignItems="center">
-            <Grid item style={{ paddingTop: '48px' }}>
-              <Typography variant='h3' className={classes.txtStyle}> This project is so new,<br></br> we are celebrating every win</Typography>
-            </Grid>
-            <Grid item style={{ padding: '24px 0px' }} >
-              <Typography variant='h6' className={classes.ptextStyle}>Let us know when you&apos;ve added #civictechindex</Typography>
-            </Grid>
-            <Grid style={{ paddingBottom: '32px' }}>
-              <Button className={classes.btnStyle}>Added to Civic Tech Index</Button>
-            </Grid>
-          </Grid>
-          <Grid container direction="row" justify="center" style={{ paddingBottom: '30px' }}>
-            <PaperComp text='Add Another Project' btext='Tag Generator' bhref='/tag-generator'/>
-            <PaperComp text='Collaborate with us' btext='Learn More' bhref='/support'/>
-          </Grid>
-        </Container>
-      </Box>
-    </>
-  )
-}
-
 const HowToUse = () => {
-  const classes = useStyles()
-  const StepComp =({ stepText,src,alt }) =>{
+  const classes = useStyles();
+  const breadCrumbLinks = [
+    { name: 'Home', href: '/home' },
+    { name: 'How to Add Your Project', href: '/join-index/how-to-add' },
+  ];
+  const StepComp = (props) => {
     return (
-      <Grid container >
-        <Grid item xs={6} style={{ padding:'8px 0px' }}>
-          <Typography variant='h6' className={classes.typoStyle}>{stepText}</Typography>
+      <Grid container>
+        <Grid item xs={6} style={{ padding: '8px 0px' }}>
+          <Typography variant='h6' className={classes.typoStyle}>
+            {props.children}
+          </Typography>
         </Grid>
-        <Grid item xs={6} style={{ padding:'8px 0px' }}>
+        <Grid item xs={6} style={{ padding: '8px 0px' }}>
           <CardMedia
             className={classes.imgStyle}
-            component="img"
-            alt={alt}
-            image={src}
-            title={alt}
+            component='img'
+            alt={props.alt}
+            image={props.src}
+            title={props.alt}
           />
         </Grid>
       </Grid>
-    )
-  }
+    );
+  };
 
   return (
     <>
       <Box className='containerGray'>
+        <Box className='containerDefault'>
+          <Container>
+            <GenericHeaderSection
```

Comment:
A quick suggestion here. Would that be nice if we keep the consistent height of the header section as other pages have? 
![image](https://user-images.githubusercontent.com/26618651/126733562-833357fe-b690-4ec6-99f9-a5e6e9ba6866.png)

Reply:
Thanks, I needed to add a prop for spacing. I have a PR up that, when merged, will take care of the height of the header section through the component GenericHeaderSection.